CHANGES.md

```diff
@@ -1,3 +1,13 @@
+Synapse vNext
+=============
+
+This patch version reverts the earlier full deprecation of delegating e-mail validation from Synapse 1.64.0rc1, which causes problems in certain deployments. It is still considered unsupported and will be fully deprecated in a timely manner in a future release.
+
+Internal Changes
+----------------
+
+- Revert ([\#13192](https://github.com/matrix-org/synapse/issues/13192)) from 1.64.0rc1. ([\#13406](https://github.com/matrix-org/synapse/issues/13406))
+
```

Comment:
github seems to have eaten my comment here, but the point is: we don't need this, because the content of `13406.misc` will automatically be added to the changelog during the release process.